multistage_geo_opt_ddec.ipynb

```diff
@@ -124,6 +125,22 @@
     "output = ipw.HTML('')"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If structure was provided through the URL\n",
+    "url = urlparse.urlsplit(jupyter_notebook_url)\n",
+    "try:\n",
+    "    structure_url = urlparse.parse_qs(url.query)['structure'][0]\n",
+    "    from ase.io import read\n",
+    "    structure_widget.structure = read(io.StringIO(structure_url), format='cif')\n",
```

Comment:
one thing we might want to think about is if we want to leave if hard coded at `CIF` or also allow `xyz`, `mol`, `pdb` etc. We could provide the format as additional url parameter.

Reply:
It is done by the eln importer, this guy is really outdated.